composer.json

```diff
@@ -23,19 +23,25 @@
         }
     ],
     "require": {
-        "php": ">=5.3.3",
-        "zendframework/zend-modulemanager": "2.*",
-        "zendframework/zend-stdlib": "2.*",
-        "zendframework/zend-servicemanager": "2.*",
-        "zendframework/zend-loader": "2.*",
-        "zendframework/zend-eventmanager": "2.*",
-        "zendframework/zend-mvc": "2.*",
+        "php": "^5.5 || ^7.0",
+        "zendframework/zend-modulemanager": "^2.7.2",
+        "zendframework/zend-stdlib": "^2.7.7 || ^3.0.1",
+        "zendframework/zend-servicemanager": "^2.7.6 || ^3.1.1",
+        "zendframework/zend-loader": "^2.5.1",
+        "zendframework/zend-eventmanager": "^2.6.3 || ^3.0.1",
+        "zendframework/zend-mvc": "^2.7.9 || ^3.0.2",
         "kriswallsmith/assetic": "~1.2.1"
```

Comment:
Not that I know of. I blindly adopted weierophinney's suggested constraints. I will upgrade to ^1.2.1 as tests are passing locally